meson.build

```diff
@@ -1,54 +1,67 @@
-#Project definition
+# Project definition
 project('performous',
         'cpp',
         version: '1.x',
-        default_options : ['cpp_std=c++17','buildtype=debugoptimized'],
+        default_options: ['cpp_std=c++17','buildtype=debugoptimized'],
```

Comment:
I think from the CI we would like a Release build instead of debug? Especially in the case of 'nightly builds'
An alternative would be to make a second nightly build with this flag override

Reply:
Yes, I think CI should have a release build rather than a debugoptimized build.
This is to be specified in semaphore.yml